support-frontend/assets/helpers/tracking/ccpa.js

```diff
@@ -0,0 +1,18 @@
+// @flow
+
+// ----- Imports ----- //
+
+import {
+  detect as detectCountry,
+  type IsoCountry,
+} from 'helpers/internationalisation/country';
+import { isTestSwitchedOn } from 'helpers/globals';
+
+// ----- Functions ----- //
+
+export const ccpaEnabled = (): boolean => {
+  const useCCPA = isTestSwitchedOn('ccpaEnabled');
```

Comment:
Why are we managing this as a test rather than a standard switch?

Reply:
It was suggested elsewhere I use `isTestSwitchedOn` but I agree a non-test switch seems more appropriate. Will set this up  now.